data_collection/gazette/spiders/ms_campo_grande.py

```diff
@@ -22,7 +23,10 @@ def start_requests(self):
         yield Request(url)
 
     def parse(self, response, sequential=0):
-        for entry in response.json()["data"]:
+        data_json = response.json()["data"]
+        if not isinstance(data_json, Iterable):
```

Comment:
**suggestion** As the content of `data` can be or a list or `False`, you could simplify it avoiding the use of `isinstance()`:

```
data_json = response.json()["data"] or []
for entry in data_json:
    ...
```
It will always return a list:

```
In [1]: response = {"data": False}

In [2]: response["data"] or []
Out[2]: []

In [3]: response = {"data": [{"entry": "value"}]}

In [4]: response["data"] or []
Out[4]: [{'entry': 'value'}]
```

Reply:
your suggestion was very good.